src/components/UI/buttons/toggleButton/ToggleButton.tsx

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import { IToggleButtonProps } from "components/UI/buttons/toggleButton/typesDef";
```

Comment:
./typesDef

Reply:
s
![Screenshot from 2022-08-15 15-19-12](https://user-images.githubusercontent.com/92886035/184635307-39bf7c17-72bd-4cc4-bbbe-bbbf9c9df614.png)
orry, as I understand this comment about non absolute import but in code this import is absolute